app/javascript/app/components/compare-ghg-chart/compare-ghg-chart-styles.scss

```diff
@@ -1,31 +1,27 @@
 @import '~styles/layout.scss';
 
-.col6 {
-  @extend %grid;
+.dropDownsLayout {
+  @include columns(6);
 
-  @include msGridColumns(1fr, 1fr);
-
-  grid-template-columns: repeat(2, 1fr);
-  margin-bottom: 10px;
+  margin-bottom: $gutter-padding;
 
   @media #{$tablet-landscape} {
-    @include msGridColumns(1fr, 1fr, 1fr, 1fr, 1fr, 1fr);
-
-    grid-template-columns: repeat(6, 1fr);
+    @include columns((2, 2, 2, 1, 3, 2));
```

Comment:
The Button group seems to have a 2 column size in the design instead of 3. Also could we fill the gap with an offset instead of more columns?

Reply:
Button group does not have a consistent grid lining up on the designs. To avoid buttons to narrow to much i've decided to  give it 3 spaces? Does it makes sense to you.  
![image](https://user-images.githubusercontent.com/6906348/37280316-b8441348-25ed-11e8-84bd-c265c3514eb6.png) . 
Agree about empty columns and offset.